.github/workflows/ci.yml

```
@@ -17,13 +17,24 @@ jobs:
             os: macos-12
```

Oops - another instance
```suggestion
            os: macos-13
```